packages/core/core/src/types.ts

```diff
@@ -253,6 +253,14 @@ export interface UserOrderGetters<ORDER, ORDER_ITEM> {
   [getterName: string]: (element: any, options?: any) => unknown;
 }
 
+export interface ReviewGetters<REVIEW> {
+  getReviewId: (review: REVIEW) => string;
+  getReviewAuthor: (review: REVIEW) => string;
+  getReviewMessage: (review: REVIEW) => string;
+  getReviewRating: (review: REVIEW) => number;
+  getReviewDate: (review: REVIEW) => string;
```

Comment:
one missing: getTotalReviews => number, Also I'm considering a more advanced rating where you can see number of people per rate. eg: 

![Screenshot 2020-09-09 at 11 56 33](https://user-images.githubusercontent.com/7943292/92584118-885afc80-f293-11ea-8840-aafdd0829c5d.png)

so i Suggest following getters:

getAvererageRating => number - that gives ou the average rate
getTotalReviews => number - the total number of reviews
getReviewRating => AgnosticRating - that gives you more structured reviews data

```ts
interface AgnosticRate {
  rate: number;
  count: number;
}

interface AgnosticRating {
  averageRate: number; // same as getAvererageRating
  totalReviews: number; // same as getTotalReviews
  rates: Rate[]; // structured rating eg: { rate:5, count 123213 } etc
}
```

now when you want to only display the average rate you can use getAvererageRating but if you need something like on the screenshot above - getReviewRating that contains everything to display the rating. 

CC @filrak

Reply:
I don;t think we need `interface AgnosticRate {
  rate: number;
  count: number;
}`

we could have just
```
 rates: { rate, count }
```

Also AgnosticRating -> AgnosticReviewsSummary ?

Reply:
@filipsobol 

getAvererageRating => number - that gives ou the average rate
getTotalReviews => number - the total number of reviews
getReviewSummary => AgnosticRate[] - that gives you more structured reviews data

interface AgnosticRate { rate: number count: number }